kernel/proto.h

```diff
@@ -83,6 +83,7 @@ BOOL check_break(void);
 UCOUNT GenericReadSft(__FAR(sft) sftp, UCOUNT n,__FAR(void) bp,
                       COUNT * err, BOOL force_binary);
 COUNT SftSeek(int sft_idx, LONG new_pos, unsigned mode);
+COUNT SftSeek2(int sft_idx, LONG new_pos, unsigned mode, unsigned long * p_result);
```

Comment:
Why?
Shouldn't it be static?

Reply:
I didn't detect that DosSeek is in the same file. I'm about to amend the commit to make this static, yes.

Reply:
Done.